aries-site/src/examples/templates/wizard/components/data.js

```diff
@@ -0,0 +1,123 @@
+import { Text } from 'grommet';
+
+export const identityProvider = [
```

Comment:
I must be blind. Where are these objects being consumed?

Reply:
Whoops. These can be removed now, thanks. I had initially added them when the demo was aligned with CCS specific steps, but removed once I was making the demo more generic again. I'll remove. Thanks!